### PR TITLE
User Story 29

### PR DESCRIPTION
**Pull request by leiyakenney**
- Updates seeds to attach merchant admin/employee to a merchant (bike_shop)
- Updates merchant dashboard view to include merchant information 
- Updates various tests with merchant users that needed to be attached to a merchant
- RSpec runs clean

### CODE REVIEW

File: app/models/merchant_user.rb [hunk 1]
```diff
@@ -0,0 +1,4 @@
+class MerchantUser < ApplicationRecord
```
Comment (MackHalliday) [inline app/models/merchant_user.rb]: Saw you deleted out a MerchantUser table. Do we still need this model as well?

Comment (leiyakenney) [reply app/models/merchant_user.rb]: Probably not. I honestly just didn't want to go crazy with deleting files. I'll delete it with my next commit.

File: app/views/merchant/dashboard/index.html.erb [hunk 2]
```diff
@@ -1 +1,7 @@
 <h1> Welcome, <%= @merchant_user.name %>!</h1>
+<br>
+<center>
+  <h3><%= @merchant.name %></h3>
```
Comment (nancylee713) [inline app/views/merchant/dashboard/index.html.erb]: At first, I thought Michael Scott should run a shop called "Mike's Bike Shop", but I realized that Michael could work for "Meg's Bike Shop" too. A little confusing, but I think this is okay :D

Comment (leiyakenney) [reply app/views/merchant/dashboard/index.html.erb]: Lmao this is called "Leiya is lazy and copied one of the shops from another page in the project -_-"

**Pull request merged**